gisutils/vector.py

```diff
@@ -207,3 +208,79 @@ def glue_lines_together(gdf, seed_id, id_col, geom_col='geometry', max_tries=Non
             success += 1
 
     return line, extra_geoms
+
+
+def line_to_point(line_geom,interval_dist):
+    """
+    Create a series of points x distance apart along a line geometry
+    """
```

Comment:
docstrings should be in the numpy doc format: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt